MinecraftClient/Protocol/Handlers/Packet/s2c/DeclareCommands.cs

```diff
@@ -29,7 +29,19 @@ public static void Read(DataTypes dataTypes, Queue<byte> packetData, int protoco
                 Parser? parser = null;
                 if ((flags & 0x03) == 2)
                 {
-                    if (protocolVersion <= Protocol18Handler.MC_1_19_2_Version)
+                    if (protocolVersion switch
+                        {
+                            Protocol18Handler.MC_1_19_Version => parserId == 50,
+                            Protocol18Handler.MC_1_19_2_Version => parserId == 50,
+                            Protocol18Handler.MC_1_19_3_Version => parserId == 50,
+                            Protocol18Handler.MC_1_19_4_Version => parserId == 50,
+                            Protocol18Handler.MC_1_20_Version => parserId == 51,
+                            Protocol18Handler.MC_1_20_2_Version => parserId == 51,
+                            _ => false
+                        }) 
+                        parser = new ParserForgeEnum(dataTypes, packetData);
+
+                    else if (protocolVersion <= Protocol18Handler.MC_1_19_2_Version)
```

Comment:
Forge's parser id changed in 1.20 and 1.20.1, while the Vanilla's didn't change a lot. I think putting it in the switch block will create another large `else if` branches for determining the version, because each one of which needs to implement the Vanilla parsers.